src/main/kotlin/com/terraformation/backend/documentproducer/api/VariableOwnersController.kt

```diff
@@ -0,0 +1,61 @@
+package com.terraformation.backend.documentproducer.api
+
+import com.terraformation.backend.api.InternalEndpoint
+import com.terraformation.backend.api.SimpleSuccessResponsePayload
+import com.terraformation.backend.api.SuccessResponsePayload
+import com.terraformation.backend.db.default_schema.ProjectId
+import com.terraformation.backend.db.default_schema.UserId
+import com.terraformation.backend.db.docprod.VariableId
+import com.terraformation.backend.documentproducer.db.VariableOwnerStore
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Schema
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PutMapping
+import org.springframework.web.bind.annotation.RequestBody
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@InternalEndpoint
+@RequestMapping("/api/v1/document-producer/projects/{projectId}/owners")
+@RestController
+class VariableOwnersController(
+    private val variableOwnerStore: VariableOwnerStore,
+) {
+  @GetMapping
+  @Operation(
+      summary = "List the owners of a project's variables.",
+      description = "Only variables that actually have owners are returned.")
+  fun listVariableOwners(@PathVariable projectId: ProjectId): ListVariableOwnersResponsePayload {
```

Comment:
I wonder if we can optionally add the ownership information into the "get variables" response for admins. It will be much more straightforward on the FE if we don't need to coalesce the two API responses.

Reply:
I didn't put it there because that endpoint doesn't take a project ID as input, so there'd be no way to know which owners to return.